docs/quickstart.md

```diff
@@ -20,32 +20,46 @@ For more details, see [the installation guide](install.md).
 
 Once you have the Red Sky Ops Tool you can initialize the manager in your cluster:
 
+<!-- @init -->
 ```sh
-$ redskyctl init
+redskyctl init
 ```
 
 ## Create a Simple Experiment
 
 Generally you will want to write your own experiments to run trials on your own applications. For the purposes of this guide we can use the simple example found in the `k8s-experiment` [repository on GitHub](https://github.com/redskyops/k8s-experiment/tree/master/examples/simple):
 
+<!-- @apply -->
 ```sh
-$ kustomize build github.com/redskyops/k8s-experiment//examples/simple | kubectl apply -f -
+kustomize build github.com/redskyops/k8s-experiment//examples/simple | kubectl apply -f -
 ```
 
 ## Run a Trial
 
 With your experiment created, you can be begin running trials by suggesting parameter assignments locally. Each trial will create one or more Kubernetes jobs and will conclude by collecting a small number of metric values indicative of the performance for the trial.
 
-To interactively create a new trial for the example experiment, run:
+To run a trial with parameters assigned, run `redskyctl suggest`:
 
+<!-- @manualSuggestion @sleep -->
 ```sh
-$ redskyctl suggest --interactive simple
+redskyctl suggest simple --assign workers=2 -A batchSize=265
 ```
 
-You will be prompted to enter a value for each parameter in the experiment and a new trial will be created. You can monitor the progress using `kubectl`:
+To interactively create a new trial for the example experiment, run:
+
+> ```
+> $ redskyctl suggest --interactive simple
+> Assignment for integer parameter 'batchSize' [128,1024]: 256
+> Assignment for integer parameter 'workers' [1,10]: 2
+> ```
+
+You will be prompted to enter a value for each parameter in the experiment and a new trial will be created.
+
+You can monitor the progress using `kubectl`:
 
+<!-- @getTrials -->
 ```sh
-$ kubectl get trials
+kubectl get trials
```

Comment:
What validates that this ran successfully?

Reply:
Good question.`mdrip` defines success as "every command exited with code 0". So this block will not validate anything other than that `kubectl get trials` runs without error. How about I change it to:

```
kubectl get trials | grep simple
```

...which will fail if nothing in the list matches?

Reply:
(There are alternative ways we could achieve that of course, but I think the `grep` technique strikes the right balance between making the test better and keeping the docs readable.)

Reply:
Why not `kubectl get trial simple`?

Reply:
Because `kubectl get` looks for an exact match, so that'll fail due to the suffix on the trial.